GitUI/GitUI.csproj

```diff
@@ -75,7 +75,7 @@
     <None Include="gituikey.snk.pfx" />
     <None Include="Resources\ChangeLog.md" />
 
-    <Content Include="Themes\dark.colors">
```

Comment:
win10default.colors is not converted, as well as .css are not read
But as a principle OK, a standard format to edit the files instead of the a custom, especially since the editor was excluded.